recipes/libtommath/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+context:
+  version: "1.3.0"
+
+package:
+  name: libtommath
+  version: ${{ version }}
+
+source:
+  url: https://github.com/libtom/libtommath/archive/v${{ version }}.tar.gz
+  sha256: 6d099e93ff00fa9b18346f4bcd97dcc48c3e91286f7e16c4ac5515a7171c3149
+
+build:
+  number: 0
+  skip:
+    - osx
+  script:
+    - if: unix
+      then:
+        - mkdir build
+        - cd build
+        - cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON ..
+        - make -j$CPU_COUNT
+        - make install
+    - if: win
+      then:
+        - mkdir build
+        - cd build
+        - cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF ..
+        - nmake
+        - nmake install
```

Comment:
Doesn't it work if you write make instead of nmake? If you can use make as a name, then you can deduplicate code here as well.